src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int BOUND = 20;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        Figure figure;
+        int index = random.nextInt(FigureName.values().length);
+        switch (FigureName.values()[index]) {
+            case CIRCLE :

```

Comment:
fix code style in all cases 
```suggestion
            case CIRCLE:

```